docs/workflow/trimming/feature-switches.md

```diff
@@ -13,6 +13,7 @@ configurations but their defaults might vary as any SDK can set the defaults dif
 | EnableUnsafeBinaryFormatterSerialization | System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization | BinaryFormatter serialization support is trimmed when set to false |
 | EventSourceSupport | System.Diagnostics.Tracing.EventSource.IsSupported | Any EventSource related code or logic is trimmed when set to false |
 | InvariantGlobalization | System.Globalization.Invariant | All globalization specific code and data is trimmed when set to true |
+| MetricsSupport | System.Diagnostics.Metrics.IsSupported | Any Metrics related code or logic is trimmed when set to false |
```

Comment:
One naming suggestion I've been making is to make the appcontext switch a possible namespace + Type + property name. That way if we ever needed to expose this in a public API, they wouldn't be inconsistent.

`System.Diagnostics.Metrics` is a namespace, which can't have properties. So I'd suggest something like:

System.Diagnostics.Metrics.Meter.IsSupported

Which makes it consistent with `System.Diagnostics.Tracing.EventSource.IsSupported`.